pages/contact.html

```diff
@@ -164,7 +170,7 @@ <h4>Developers:</h4>
 		<li>Guilherme Felipe de C. G. da Silva (guilhermefelipecgs)</li>
 		<li>Hanif Bin Ariffin (hbina)</li>
 		<li>Haoyu Qiu (timothyqiu)</li>
-		<li>HP van Braam (hpvb)</li>
```

Comment:
I believe this is how HP prefers their name to be written, so I wouldn't change this line.